muhtalhakhan/Hacktoberfest2024
# Implemented a simple GET endpoint that returns 'Hello, World\!' message

### Description
This PR implements a basic GET endpoint in the Express server that responds with a "Hello, World!" message when accessed at the root URL (`/`).

### Changes:
- Added a simple GET route in the Express application that returns the message "Hello, World!".
- Configured the server to listen on a port defined by the `.env` file or default to port 3000.

### How to Test:
1. Clone the repository and check out the branch `feature/hello-world-endpoint`.
2. Run `npm install` to install the dependencies.
3. Start the server by running `npm start`.
4. Open a browser or use `curl` to access `http://localhost:<PORT>` and verify that it returns "Hello, World!".

### Additional Information:
- Make sure your `.env` file includes a `PORT` variable to specify the port on which the server will run.

### Fixes:
- This PR fixes issue #24.
